### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by zakiransari003** [datasciencecompleteguide/Machine-Learning#27] [General]
error: OpenCV(4.5.3) :-1: error: (-5:Bad argument) in function 'inRange'
> Overload resolution failed:
>  - src is not a numpy array, neither a scalar
>  - Expected Ptr<cv::UMat> for argument 'src'

Resolve it please!!!!!!!!!!!!!!!

**Comment by datasciencecompleteguide** [comment]
> error: OpenCV(4.5.3) 👎 error: (-5:Bad argument) in function 'inRange'
> 
> > Overload resolution failed:
> > 
> > * src is not a numpy array, neither a scalar
> > * Expected Ptrcv::UMat for argument 'src'
> 
> Resolve it please!!!!!!!!!!!!!!!



Somewhere you have missed the argument:
For example:

def function(): // I don't have any variable to recieve the argument
       cap.set(7, 200)
       cap.set(5, 600)

while(vid.isOpened()):
         _, frame = vid.read()
         resized_image = function(cap) //Here I am passing the argument to the function
         final_image = cv.Canny(frame, 0, 255) //The real input should be the "resized_image" instead of that i have passed the "frame" itself

Hope you would find the mistake in your code after this.
Otherwise please share your code with us, so that we can resolve your issue more specifically!
Thanks.

**Comment by zakiransari003** [comment]
Thanks...... Resolved!!!
